kernelflinger.c

```diff
@@ -479,6 +480,22 @@ static enum boot_target check_command_line(VOID)
 			continue;
 		}
 
+		if (!StrCmp(argv[pos], L"-l")) {
+			pos++;
+			if (pos >= (argc + 1)) {
+				error(L"-l requires output kernelflinger log to the file");
+				goto out;
+			}
+			if (!log_output) {
```

Comment:
What is the point of all this logic with the `log_output` variable ?
Same question for `installer.c`

Reply:
Already deleted the log_output variable.